source/Halibut.Tests/Timeouts/ReceiveResponseTimeoutTests.cs

```diff
@@ -92,5 +93,37 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
                         "A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond");
             }
         }
+        
+        [Test]
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testWebSocket: false)]
+        public async Task WhenRpcExecutionIsWithinReceiveResponseTimeout_ButDataStreamDataIsDelayed_ThenTimeoutShouldOccur(ClientAndServiceTestCase clientAndServiceTestCase)
+        {
+            var halibutTimeoutsAndLimits = new HalibutTimeoutsAndLimits
+            {
+                // Ensure execution time does not trigger timeouts
+                TcpClientReceiveResponseTimeout = TimeSpan.FromHours(1),
+                TcpClientReceiveResponseTransmissionAfterInitialReadTimeout = TimeSpan.FromMilliseconds(100)
```

Comment:
Maybe bump this timeout, since it is low enough that on TC it could be the reading of the JSON response that trips it.

Reply:
I bumped it to 1 second. I wasn't sure what number you had in mind.